.github/workflows/deploy.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+      - name: Install and configure poetry
+        run: python -m pip install poetry
+      - name: Install python packages
+        run: python -m poetry install
```

Comment:
I don't remember exactly why and when, but the doc on Confluence says to use `--no-root` for the first poetry install, and then call run-ci with `--use-tanker deployed`

Is that not needed when deploying?

Reply:
I think `--use-tanker deployed` is only related to finding `sdk-native` source code/.so/.a. I don't remember about `--no-root` though

Reply:
Arn't these options for `poetry run run-ci.py install`?

Reply:
Long story short: this applies to the sdk-python, because poetry install will try to compile native. Non-applicable here!